tdrs-backend/plg/alertmanager/alertmanager.yml

```diff
@@ -43,7 +43,7 @@ route:
     - matchers:
         - alertname=~"UpTime"
       receiver: dev-team-emails
-      group_wait: 30m
+      repeat_interval: 24h
```

Comment:
bonus change: alerts should only fire once per 24 hours for active alarms (limiting how many emails are sent during an incident)